package.json

```diff
@@ -13,13 +13,13 @@
     "url": "https://github.com/GPII/gpii-mail-test"
   },
   "dependencies": {
-    "infusion": "2.0.0-dev.20151211T025229Z.eabb4fc",
+    "infusion": "git://github.com/amb26/infusion.git#81ef49cdc0e78a96549c5195d9ffd42604087096",
```

Comment:
I've just rolled 2.0.0-dev.20160505T142427Z.63b8a53 which contains all my latest fixes - see if this works for you. To speed the process a bit, you can self-push an update to gpii-express if it just consists of an update to this version and all the tests pass.